### ISSUE TITLE
Local Meetups

### ISSUE THREAD

**Issue by simonv3** [opensourcedesign/organization#58]
Can we set up a "local chapters" flow in the same way that NodeSchool does it? 

1. Either here or in an other repository people can plan to create local meetups. 
2. When they've gotten a small group together (1+) who are interested in organizing something they request a repository
3. A repository gets created (manually for now, automated in the future?) and a basic site gets set up for the local meetup with name and city. Eg, the repository name could be "berlin". 
4. The creators of the admin have full control of the repository for that event.
5. They use that repository to plan their events in the locale. 

Inspired by #57.

**Comment by elioqoshi** [comment]
+1 to this.

Berlin and Tirana seems already going on.
I do think have a repo/site for every chapter is overengineering for now, but yeah.

**Comment by jancborchardt** [comment]
Yup, sounds good! I do agree that we should try and reduce repositories rather than create more though.

There could be one event posting per city in the events repo and one corresponding issue in case there are questions. That’s a small start for now which doesn’t spread things out again.

**Comment by Xaviju** [comment]
Yes please! I'd love to create a local event in Madrid, Spain. Should I create an issue to encourage my fellow locals to join the group?

**Comment by simonv3** [comment]
@jancborchardt sounds good. 

So, one issue per location and a series of local meetup groups on the events page? 

@Xaviju, yes please go ahead!

**Comment by victoria-bondarchuk** [comment]
@Xaviju Great idea, will join you soon!

**Comment by bnvk** [comment]
Big +1 to this idea, I love nodeschool style organizing. I guess if you want to 
start one, just create a `/city-name` repo and then make sure it's linked to
via /events page on site and repo?

@Xaviju @victoria-bondarchuk if you'd like to be added to the Doodle invite
(for Summit date choosing) can you send me your email addresses somewhere?

**Comment by jancborchardt** [comment]
@bnvk check my comment above ;)

> Yup, sounds good! I do agree that **we should try and reduce repositories rather than create more though**.
> There could be one event posting per city in the events repo and one corresponding issue in case there are questions. That’s a small start for now which doesn’t spread things out again.

**Comment by jancborchardt** [comment]
I added some Twitter lists and Github groups so people can find Open Source Designers nearby!
- [Open Source Design lists on Twitter](https://twitter.com/opensrcdesign/lists)
- [Open Source Design teams on Github](https://github.com/orgs/opensourcedesign/teams/) (only visible to organization members)

Let’s add more! (@simonv3 wasn’t sure which place to create for you, sorry. ;)

**Comment by jancborchardt** [comment]
I added some Twitter lists and Github groups so people can find Open Source Designers nearby!
- [Open Source Design lists on Twitter](https://twitter.com/opensrcdesign/lists)
- [Open Source Design teams on Github](https://github.com/orgs/opensourcedesign/teams/) (only visible to organization members)

Spread the word by retweeting our tweet about it! https://twitter.com/opensrcdesign/status/840323588666335232

Let’s add more! (@simonv3 wasn’t sure which place to create for you, sorry. ;)

**Comment by simonv3** [comment]
So my issue with the OSD lists on twitter (and I mentioned this in #opensourcedesign) is that they don't self organize, someone has to add these people to these lists. And we just figured out how to automatically add them for GitHub. 

I think teams on GitHub is a great idea, but what does it add? Just awareness? Can people add themselves to these teams?

**Comment by jancborchardt** [comment]
@simonv3 people can add themselves to these teams on Github, yes. The teams I created on both Twitter and Github strived to have the same people on both platforms. (So please if you add them on Twitter – follow them from our account and also add them to our Github org in the specific team.)

Sure this is something we can do better, but this is a minimum viable solution when we travel and look for open source designers to collaborate with. Or a first way for people to be aware of other open source designers around.

**Comment by jancborchardt** [comment]
@whitef0x0 mentioned interest in starting a meetup in San Francisco! :tada: 

There’s some local folks like @erinjo @neynah @objectadjective @sophiakc @jina @Boriss @vdepizzol @behdad @eff-soraya – also see the SF people on our Twitter list: https://twitter.com/opensrcdesign/lists/san-francisco/members and Github team: https://github.com/orgs/opensourcedesign/teams/san-francisco :)
